From 8449c8623d6208b77e76596e8b10250817d3e8ab Mon Sep 17 00:00:00 2001
From: Adam Jackson <ajax@redhat.com>
Date: Mon, 19 Aug 2019 14:27:54 -0400
Subject: [PATCH xserver 11/15] miext/sync: Fix needless ABI change

The initialized field was added in:

    commit 82f01ad7869e3f2be51e41a8246dab5982bbc36a
    Author: Alex Goins <agoins@nvidia.com>
    Date:   Wed Apr 10 13:48:02 2019 -0500

        xsync: Add resource inside of SyncCreate, export SyncCreate

But it added this field not at the end of SyncObject. It may not have
been _usefully_ possible to create those from another extension prior to
that commit, but that's still an ABI-incompatible change.

(cherry picked from commit 194ba38728b34a76885275dae153057c8afaf72e)
---
 miext/sync/misyncstr.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/miext/sync/misyncstr.h b/miext/sync/misyncstr.h
index 2a6e84a96..ea48a4519 100644
--- a/miext/sync/misyncstr.h
+++ b/miext/sync/misyncstr.h
@@ -43,8 +43,8 @@ struct _SyncObject {
     struct _SyncTriggerList *pTriglist; /* list of triggers */
     XID id;                     /* resource ID */
     unsigned char type;         /* SYNC_* */
-    Bool initialized;           /* FALSE if created but not initialized */
     Bool beingDestroyed;        /* in process of going away */
+    Bool initialized;           /* FALSE if created but not initialized */
 };
 
 typedef struct _SyncCounter {
-- 
2.21.0

